Update from GlobalQSS

victorsuarezis opened the pull request.

Commit:
Facturación Electrónica

CarlosRuiz-globalqss committed:
Add plugin org.globalqss.idempiere.LCO.firstname - a simpler subset of org.globalqss.idempiere.LCO.detailednames

CarlosRuiz-globalqss committed:
Add plugin org.globalqss.idempiere.LCO.firstnamecontact - a simple plugin to add first/last name to user/contact table

CarlosRuiz-globalqss committed:
Sincronizar traducción español Colombia con release-9

CarlosRuiz-globalqss committed:
Corregir error ortográfico en es_CO

j2garcia committed:
[#3174] QSS - FE Documento Soporte DIAN - Version 1.1 Beta

j2garcia committed:
[#3174] QSS - FE Documento Soporte DIAN - Version 1.1 Beta

j2garcia committed:
[#3174] QSS - FE Documento Soporte DIAN - Version 1.1 Beta

CarlosRuiz-globalqss committed:
Move electronic invoice to https://github.com/globalqss/globalqss-idempiere-lcofe

CarlosRuiz-globalqss committed:
Move to release-9

CarlosRuiz-globalqss committed:
Correcciones ortográficas

CarlosRuiz-globalqss committed:
IDEMPIERE-5360 Payment > REVERSE ACCRUAL posts payment reversal allocation to GL Write-off instead of Withholding Tax

etantg committed:
IDEMPIERE-5360 Payment > REVERSE ACCRUAL posts payment reversal allocation to GL Write-off instead of Withholding Tax

Fix issue with organization

argenis97 committed:
Move to release 10 (#5)

* Move to release 10

* Fix Relative Path

CarlosRuiz-globalqss committed:
Cleanup

aguerrave reviewed (approved):
aplicando traducciones

CarlosRuiz-globalqss committed:
Add SysConfig to define if first or last name are mandatory

CarlosRuiz-globalqss committed:
IDEMPIERE-5727 LCO: Error when reverse correct a payment

CarlosRuiz-globalqss committed:
Añadir dos nuevos campos del formato 2276v4

CarlosRuiz-globalqss committed:
Fix problem with rounding of withholdings in different currency

- When the currency of the invoice is different from the currency of the accounting use costing precision to avoid problems with rounding

CarlosRuiz-globalqss committed:
Prepare for release-11

CarlosRuiz-globalqss committed:
Sync with release-11

CarlosRuiz-globalqss committed:
move back to p2.site

CarlosRuiz-globalqss committed:
- synchronize es_CO translation with r13

CarlosRuiz-globalqss committed:
- Synchronize with release-13

CarlosRuiz-globalqss committed:
- sync es_CO

CarlosRuiz-globalqss committed:
- make consistent
